endpoints/openrtb2/auction.go

```diff
@@ -456,14 +457,14 @@ func validateNativeContextTypes(cType native.ContextType, cSubtype native.Contex
 		return fmt.Errorf("request.imp[%d].native.request.context is invalid. See https://iabtechlab.com/wp-content/uploads/2016/07/OpenRTB-Native-Ads-Specification-Final-1.2.pdf#page=39", impIndex)
 	}
 	if cSubtype < 0 {
-		return fmt.Errorf("request.imp[%d].native.request.contextsubtype is invalid. See https://iabtechlab.com/wp-content/uploads/2016/07/OpenRTB-Native-Ads-Specification-Final-1.2.pdf#page=39", impIndex)
+		return fmt.Errorf("request.imp[%d].native.request.contextsubtype value can't be less than 0. See https://iabtechlab.com/wp-content/uploads/2016/07/OpenRTB-Native-Ads-Specification-Final-1.2.pdf#page=39", impIndex)
 	}
 	if cSubtype == 0 {
 		return nil
 	}
 
 	if cSubtype >= 100 {
-		return fmt.Errorf("request.imp[%d].native.request.contextsubtype is invalid. See https://iabtechlab.com/wp-content/uploads/2016/07/OpenRTB-Native-Ads-Specification-Final-1.2.pdf#page=39", impIndex)
+		return fmt.Errorf("request.imp[%d].native.request.contextsubtype can't be greater than or equal to 100. See https://iabtechlab.com/wp-content/uploads/2016/07/OpenRTB-Native-Ads-Specification-Final-1.2.pdf#page=39", impIndex)
```

Comment:
The linked page suggests that there are some valid values over 100, though I suppose not currently in our implementation.  Maybe the old message is safer here!?

Reply:
I changed it to that to indicate that we expect the values to be only below 100. Consider, if some user uses a value above 100 and gets back the old error message, they wouldn't know why exactly are they getting back an error. According to them it should be valid, given the 
IAB spec but with the new error message it will be clear to them that the reason for the error is because we don't support a value that is greater than or equal to 100. Hope that makes sense!

Reply:
It does.  My point was that the page mentioned isn't consistent with the new statement:  the page says there are possible valid values above 100.  Seems potentially confusing, though not a big deal.

Reply:
Yeah, your point is totally valid but I just think that the old message would be much more confusing than the new one as the new one at least indicates that we don't support a value greater than or equal 100.

What do you think about the error message to say: `request.imp[%d].native.request.contextsubtype value greater than or equal to 100 isn't supported`? Do you think that would be more useful? If not, I am happy to consider any other suggestions that you think would be easier for the user to understand what went wrong.

Reply:
I don't feel very strongly about any particular solution.  My slight preference would be to use your updated text but to remove the reference to the page from the error message.

Reply:
Actually, I pondered a little more over this after you pointed it out and wondered if that check should be actually checking for a value greater than 500 rather than greater than 100 and confirmed with @hhhjort that it should actually check for value greater than 500. Will update the PR accordingly.

Thanks for pointing it out @josephveach!